sealedenumprocessor/src/test/kotlin/com/livefront/sealedenum/JavaBaseClasses.kt

```diff
@@ -0,0 +1,161 @@
+package com.livefront.sealedenum
+
+import org.intellij.lang.annotations.Language
+
+interface JavaBaseClassesInterface1
```

Comment:
Agreed, which is why I kinda just kept adding interfaces to cover more edge cases. I'll make them named more explicitly!

Reply:
Alright, the main reason why names were getting annoying was because of avoiding conflicts with all of the other tests. In fa5ba7e I bit the bullet and moved all of the compilation tests into loosely related subpackages.

This allowed me to rename these interfaces in the newly named `kitchensink` tests to simply `KotlinInterface*` without fear of running into that same name elsewhere in 285bdbe.

Sort of unrelated, but I added b0f522e to make `getSourceFile` work for both `.kt` and `.java` files as well.